src/game/clients/CClientMsg_AOSTooltip.cpp

```diff
@@ -112,6 +112,13 @@ bool CClient::addAOSTooltip(CObjBase * pObj, bool fRequested, bool fShop)
 
                 pObj->AddPropsTooltipData(pObj);
 			}
+			
+			if (IsTrigUsed(TRIGGER_CLIENTTOOLTIP_SENT) || (pItem && IsTrigUsed(TRIGGER_ITEMCLIENTTOOLTIP_SENT)) || (pChar && IsTrigUsed(TRIGGER_CHARCLIENTTOOLTIP_SENT)))
+			{
+				CScriptTriggerArgs args(pObj);
+				args.m_iN1 = fRequested;
+				iRet = pObj->OnTrigger("@ClientTooltip_Sent", this->GetChar(), &args); //Save to return on iRet to make sure return value doesn't stuck the boolean.
```

Comment:
Tip: it's faster to use the enum instead of the string

Reply:
Yeah, but I just copied ClientTooltip trigger and it uses string, so I haven't changed the way, because I haven't sure about if someone doing it for some reason, but I can change it to ITRIG enum.

Reply:
Hmm seems like CScriptObj only support lpctstr because when I change it to ITRIG_CLIENTTOOLTIP_SENT it gives that error

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2664	'TRIGRET_TYPE CScriptObj::OnTrigger(lpctstr,CTextConsole *,CScriptTriggerArgs *)': cannot convert argument 1 from 'ITRIG_TYPE' to 'lpctstr'	spheresvr	C:\Users\user\Documents\GitHub\Source-X\src\game\clients\CClientMsg_AOSTooltip.cpp	120
```

Reply:
You're right, there are no better alternatives here